examples/demo-app/src/cloud-providers/fsq-studio/fsq-studio-icon.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import {Icons} from 'kepler.gl/components';
+import PropTypes from 'prop-types';
+
+const style = {
+  background: 'black'
+};
+export default class UnfoldedIcon extends Component {
```

Comment:
FoursquareIcon? (Nit: If we stop using default exports we won't be making such mistakes)